scripts/verify-promote

```diff
@@ -104,11 +106,17 @@ banner "Verify tarball install and that 'sf update $NEW_CHANNEL' version matches
 # If you are testing this locally, you'll need to use a tarball built for your OS
 #   There is a VERSIONS_URL for M1 macs above that you can uncomment
 #   Other URLs can be found here: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_olderversions
-OLDER_VERSION_URL=$(echo "$VERSION_CONTENT" | jq 'to_entries[7]' | jq -r '.value')
+OLDER_VERSION_URL=$(echo "$VERSION_CONTENT" | jq 'to_entries[15]' | jq -r '.value')
 echo "Downloading old cli version from: $OLDER_VERSION_URL"
 
 if [[ -d "$HOME/sf" ]]; then
-  echo "!! $HOME/sf directory already exists. Delete it and try again." && exit 1
+  # If the CI env var is set, delete the directory and continue
+  if [[ -n "$CI" ]]; then
+    echo "!! $HOME/sf directory already exists. Deleting it and continuing."
+    rm -rf "$HOME/sf"
+  else
+    echo "!! $HOME/sf directory already exists. Delete it and try again." && echo 1
+  fi
```

Comment:
This was preventing any of the `retry` logic from making it past this step. Now we cleanup that dir in CI